src/services/anchor-service.ts

```diff
@@ -236,9 +216,10 @@ export class AnchorService {
 
       await this.requestRepository.updateRequests({ status: RS.READY }, readyRequests)
```

Comment:
Added comment but to repeat here: since the expiration of ready requests are determined by their "updated_at" field, update the requests again to indicate that a new anchor event has been emitted